dataverse/orgsvc/C#-NETCore/ElasticTables/ElasticTableSamples/ElasticTableSamples/Program.cs

```diff
@@ -0,0 +1,463 @@
+
+            // Capture the session token
+            sessionToken = response.Results["x-ms-session-token"].ToString();
+
+            return response.RecordCreated;
+        }
+
+        private static void DeleteRecord(IOrganizationService organizationService, Guid contosoSensorDataId, string deviceId, string sessionToken)
+        {
+            Console.WriteLine($"Deleting {"contoso_sensordata"} record.. \n");
+            DeleteRequest request = new DeleteRequest
+            {
+                Target = new EntityReference("contoso_sensordata", contosoSensorDataId),
+                ["partitionId"] = deviceId,
+                ["SessionToken"] = sessionToken
+            };
+
+            organizationService.Execute(request);
+            Console.WriteLine($"Deleted {"contoso_sensordata"} record.\n");
+        }
+
+        private static void DeleteRecordWithAlternateKey(IOrganizationService organizationService, Guid contosoSensorDataId, string deviceId, string sessionToken)
+        {
+            Console.WriteLine($"Deleting {"contoso_sensordata"} record with alternate key.. \n");
+            KeyAttributeCollection keyAttributeCollection = new()
+            {
+                ["contoso_sensordataid"] = contosoSensorDataId,
+                ["partitionid"] = deviceId
+            };
+
+            DeleteRequest request = new DeleteRequest
+            {
+                Target = new EntityReference("contoso_sensordata", keyAttributeCollection),
+                ["SessionToken"] = sessionToken
+            };
+
+            organizationService.Execute(request);
+            Console.WriteLine($"Deleted {"contoso_sensordata"} record with alternate key.\n");
+        }
+
+        private static List<Guid> CreateMultipleRecords(IOrganizationService organizationService, string deviceId)
+        {
+            int numberOfRecords = Settings.NumberOfRecords;
+            List<Guid> createdRecordIds = new List<Guid>();
+
+            // Create a List of entity instances.
+            Console.WriteLine($"Preparing {numberOfRecords} records to create..\n");
+            List<Entity> entityList = new();
+
+            // Populate the list with the number of records to test.
+            for (int i = 0; i < numberOfRecords; i++)
+            {
+                Entity entity = new Entity("contoso_sensordata")
+                {
+                    Attributes =
+                    {
+                        { "contoso_sensortype", $"Humidity" },
+                        { "partitionid", deviceId }
+                    }
+                };
+
+                EnergyConsumption energyConsumption = new()
+                {
+                    Voltage = i,
+                    VoltageUnit = "Volts",
+                    Power = i + 1,
+                    PowerUnit = "Watts"
+                };
+
+                entity["contoso_energyconsumption"] = JsonConvert.SerializeObject(energyConsumption);
```

Comment:
Can you add a comment to call out that `contoso_energyconsumption` is a string column with JSON format?